frontends/ol-components/src/components/SearchInput/SearchInput.tsx

```diff
@@ -54,35 +56,15 @@ const muiInputProps = { "aria-label": "Search for" }
 
 const SearchInput: React.FC<SearchInputProps> = (props) => {
   const { onSubmit, value } = props
-  const posthog = usePostHog()
-  const { POSTHOG } = APP_SETTINGS
+  const event = {
+    target: { value },
+    preventDefault: () => null,
+  }
 
-  const handleSubmit = useCallback(
```

Comment:
I moved the posthog stuff out of `ol-components` and into the application, since ol-components is generally styling / UI focused.